### ISSUE TITLE
Editing multiple entities together

### ISSUE THREAD

**Issue by Adambean** [wootguy/bspguy#11]
Would it be in scope to allow for multiple entities, selected by holding CTRL while clicking, to be edited together?

The obvious difficulty is handling differences between the entities. I would suggest:

- For keys where only the value differs, background colour the value text box red, and show the text as "(unchanged)" as per VHE.
- For keys that may not exist across all selected entities highlight the key text box red, but keep the text as the key's name.
- For flags I think this would need to use the 3rd grey state where the value differs between entities.

This becomes more complicated again when editing multiple entities of different classes. I think that for a first step it would be acceptable to not permit selecting multiple entities of different classes, leaving that for a step 2 after more thought.

**Comment by wootguy** [comment]
Yep, hammer-like selection is planned to come after undo/redo, and whenever I get back to working on this.
